### PR TITLE
w01d03

### PR DESCRIPTION
**Pull request by annieredmond1**
comfort: 4
completeness: 4

I could not get the effect to work so I just picked different things - it was super frustrating :)

### CODE REVIEW

**Comment by annieredmond1** [comment]
Travis - can you please look at my code?  jQuery that doesn't move the picture around works, but if I try to move it (bounce or explode, etc.) it won't work.  The TA thought that I might not have the right library??

Thank you!!!!!!
Annie

**Comment by tgaff** [comment]
ok, will do!

**Comment by tgaff** [comment]
@annieredmond1 .animate is a bit more complicated.  It's for designing a complicated animation when one of the simple ones just isn't enough.  That means you have to pass it a lot of details about what you want.

File: annieredmond1/script.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+$(document).ready(function() {
+	$('.one').on('click', function() {
+		$('.dog').hide("explode");
+	});
+	$('.two').on('click', function() {
```
Comment (tgaff) [inline annieredmond1/script.js]: try:
```js
	$('.two').on('click', function() {
		$('.dog').animate({
        width: "70%",
        opacity: 0.4,
        marginLeft: "0.6in",
        fontSize: "3em",
        borderWidth: "10px"
      }, 1500)
````

Comment (tgaff) [inline annieredmond1/script.js]: the 1500 there is the duration of the transition

Comment (tgaff) [inline annieredmond1/script.js]: http://api.jquery.com/animate/

**Comment by tgaff** [comment]
:tulip:

**Pull request merged**